ModularTegustation/delivery_doors.dm

```diff
@@ -14,18 +14,25 @@
 	. = ..()
 	address = "[x]-[y]"
 	name += " ([address])"
+	GLOB.delivery_doors += src
+
+/obj/structure/delivery_door/Destroy()
+	. = ..()
+	GLOB.delivery_doors -= src
```

Comment:
For Destroy() please try to always put code at the beginning and end it with a `return ..()`, its generally how its handled with it
```suggestion
	GLOB.delivery_doors -= src
	return ..()
```